docs/changelog.md

```diff
@@ -16,6 +16,9 @@ tocMaxDepth: 2
 
 - Fixed resolving scalar values in asserts.
 
+### Deprecation
```

Comment:
question: should we add an admonition here that we will still support the old "openapi" naming in commands? In general, we should announce this change somewhere, add a note that we've updated all references to "OpenAPI CLI" in the docs to "Redocly CLI", and any other relevant info.

Reply:
I think we should.

Reply:
@skadinna can you help with it?

Reply:
I made some changes here: https://github.com/Redocly/openapi-cli/pull/672/commits/22bf9d612d350d1d4d60ba6f4e4ca87baac0c193